sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_receiver_async.py

```diff
@@ -454,11 +456,13 @@ async def receive_deferred_messages(self, sequence_numbers):
         messages = await self._mgmt_request_response_with_retry(
             REQUEST_RESPONSE_RECEIVE_BY_SEQUENCE_NUMBER,
             message,
-            handler
+            handler,
+            timeout=timeout
         )
         return messages
 
-    async def peek_messages(self, max_message_count=1, sequence_number=0):
+    async def peek_messages(self, max_message_count=1, sequence_number=0, timeout=None):
```

Comment:
hmm these should all probably be keyword-only args...

Reply:
While I could cave for timeout, and MAYBE sequence number, I'm... of the mind that the former two are core enough to the happy-path function of this method that they should be surfaced.

Reply:
(seems like you two have reached an agreement ^^, keeping `max_message_count` and moving the remaining two into kwargs sounds good to me)

Reply:
sequence_number and timeout are moved into kwargs